lib/remap-functions/src/encode_json.rs

```diff
@@ -0,0 +1,154 @@
+use remap::prelude::*;
+
+#[derive(Clone, Copy, Debug)]
+pub struct EncodeJson;
+
+impl Function for EncodeJson {
+    fn identifier(&self) -> &'static str {
+        "encode_json"
+    }
+
+    fn parameters(&self) -> &'static [Parameter] {
+        &[Parameter {
+            keyword: "value",
+            accepts: |_| true,
```

Comment:
Although it would encode just fine as a String, we don't want to accept regular expressions here. It's not a huge issue, but we just try to keep regexes separate and only allow them where they can actually be used as regexes.

```suggestion
            accepts: |v| !matches!(v, Value::Regex(_))
```

Reply:
But regex still can be accepted as nested values... should we still disable regex on the top-level?

Reply:
Good point. No, I think this is fine.